### ISSUE TITLE
Blank Canvas: Edit Link Not Available on Posts or Pages on Site Front End

### ISSUE THREAD

**Issue by edequalsawesome** [Automattic/themes#3167]
## Steps to replicate

* Activate Blank Canvas
* Visit any post or page on your site while logged in to WordPress.com

## Result

The Edit link isn't available on the front-end of the site:

[![Edit link missing on a screenshot of Blank Canvas](https://d.pr/i/lJ0qUT+)](https://d.pr/i/lJ0qUT)

## Expected

The Edit link would be available, similarly to how it appears in Seedlet:

[![Edit link highlighted in a screenshot of Seedlet](https://d.pr/i/QOOjTE+)](https://d.pr/i/QOOjTE)

**Comment by kjellr** [comment]
I've always felt weird about the Edit link in general... it seems like something that should just be handled by the admin bar, and not placed in the page itself. It is in the admin bar by default for .org installs, but for some reason, WP.com hides this link. 😕

In any case, I'm not sure it's something we should add for this theme. Since the theme doesn't display post meta on its own, the edit link won't really "fit in" anywhere by default, and instead be a weird floating extra element:

No Edit Link|With Edit Link
---|---
![Screen Shot 2021-02-03 at 8 26 11 AM](https://user-images.githubusercontent.com/1202812/106753553-cddce500-65f9-11eb-9bf4-ef283a84079e.png)|![Screen Shot 2021-02-03 at 8 28 11 AM](https://user-images.githubusercontent.com/1202812/106753550-cddce500-65f9-11eb-99d9-73cc7095bda3.png)

We could try custom-designing a different placement for it (Maybe in the footer on single posts/pages?) But I'm not totally sure it's worth it. 

cc @melchoyce for thoughts if you have them. Also @ianstewart since I recall you discussing the edit link in the context of FSE once, and this is kind of similar.

**Comment by melchoyce** [comment]
Somewhat less of an issue on .org, IMO, where there's an edit link in the admin bar that I'm always drawn to. WP.com has the edit button in the front-end toolbar in the lower right, but I always seem to forget about that one/skip passed it for some reason? I think it's probably worth seeing how many folks report this before making a decision.

**Comment by kjellr** [comment]
Oh right! I forgot about that link:

![Screen Shot 2021-02-03 at 4 00 27 PM](https://user-images.githubusercontent.com/1202812/106808858-f33d1380-6638-11eb-9ae8-672037259361.png)

> I think it's probably worth seeing how many folks report this before making a decision.

Makes sense, especially since both users on .com and .org have edit links available in other UI.

**Comment by ianstewart** [comment]
> Also @ianstewart since I recall you discussing the edit link in the context of FSE once, and this is kind of similar.
>  the edit link won't really "fit in" anywhere by default, and instead be a weird floating extra element:

I would add it in places like …

https://github.com/Automattic/themes/blob/34522db0dd3d61a5d00d412661e6ae23deec0a1a/blank-canvas/template-parts/content/content-singular.php#L46

… just below "the content".

I think it should be there for consistency on all posts and pages as users will expect it to be just below the content for quick-edits.

(I'd also like to see a better edit button on the front-end similar to dot org. Also, only tangentially related but there could/should perhaps be gutenberg issue related to that button. Where does it go? The site editor? The post/page editor?)

**Comment by kathrynwp** [comment]
I agree with Ian, could we please be consistent? No other themes on WP.com hide the front-end Edit link, and the parent theme, Seedlet, also doesn't hide the Edit link. When we tested this theme in our team hangout, it was super disconcerting not to have the link. I don't think we should be forcing users to figure out where an alternative link can be found when every other theme has a front-end Edit link, when we can easily remove that cognitive burden.

**Comment by ianstewart** [comment]
> WP.com has the edit button in the front-end toolbar in the lower right, but I always seem to forget about that one/skip passed it for some reason?

It's my understanding most users skip/miss this button.

**Comment by kathrynwp** [comment]
Also noting that:

The action bar shown [in this screenshot](https://github.com/Automattic/themes/issues/3167#issuecomment-772819907) is **not** present on Atomic sites, nor is the Edit link in the admin bar:
![Post_16_-_TRFIT_Supplmenentos_and_Slack_____Threads___A8C](https://user-images.githubusercontent.com/7662449/106924333-f8539e80-66dc-11eb-9bec-14cd491aada8.jpg)

The Edit link is already present below posts on the Posts page in this theme. They are only missing on individual posts and pages. 

![TRFIT_Supplmenentos_-](https://user-images.githubusercontent.com/7662449/106924450-14574000-66dd-11eb-8a9a-274cb9ef7aa6.jpg)

**Comment by kjellr** [comment]
> The action bar shown in this screenshot is not present on Atomic sites, nor is the Edit link in the admin bar:

Oh wow, thanks for the heads up, @kathrynwp. I wish we were consistent about this! 😭
